pkg/typesystem/typesystem.go

```diff
@@ -398,7 +398,7 @@ func (t *TypeSystem) TTUCanFastPath(objectType, computedRelation, userType strin
 
 	terminalRelations := t.GetTerminalRelations(objectType, computedRelation, userType)
 
-	return len(terminalRelations) > 0
```

Comment:
Doesn't the check of line `393` have to be checking against 1 also?

Reply:
Yes, good catch. I'm going to change that.